scripts/drtprod

```diff
@@ -109,7 +109,34 @@ tags:
 - team:drt
 EOF"
 
-    roachprod ssh ${cluster} -- "sudo tee /etc/datadog-agent/conf.d/cockroachdb.d/conf.yaml > /dev/null << EOF
+    case $cluster in
+      "workload-chaos")
+        roachprod ssh ${cluster} -- "sudo tee /etc/datadog-agent/conf.d/openmetrics.d/conf.yaml > /dev/null << EOF
+---
+init_config:
+
+instances:
```

Comment:
Are there many metrics exported from these endpoints? I ask because these metrics will be billed as custom metrics [per the documentation](https://docs.datadoghq.com/integrations/openmetrics/) and a high volume of custom metrics can have unintended cost implications. If there's a reference that will list the metrics exported from these endpoints we could link it here as a comment.

Reply:
We're talking about on the order of magnitude, ~50 metrics per endpoint. That seems like it's above our default allocation, so i'll bring this down a little by eliminating the go_* ones that we don't care about. Thanks for pointing that out!